doc/cli/npm-install.md

```diff
@@ -3,15 +3,23 @@ npm-install(1) -- Install a package
 
 ## SYNOPSIS
 
-    npm install (with no args in a package dir)
-    npm install <tarball file>
-    npm install <tarball url>
-    npm install <folder>
-    npm install [@<scope>/]<name> [--save|--save-dev|--save-optional] [--save-exact]
+    npm install (with no args, in package dir)
+    npm install [@<scope>/]<name>
     npm install [@<scope>/]<name>@<tag>
     npm install [@<scope>/]<name>@<version>
     npm install [@<scope>/]<name>@<version range>
-    npm i (with any of the previous argument usage)
+    npm install <tarball file>
+    npm install <tarball url>
+    npm install <folder>
+
+    alias: npm i
+    common options: [--save|--save-dev|--save-optional] [--save-exact]
+
+    Can specify one or more: npm install ./foo.tgz bar@stable /some/folder
+
+    If no argument is supplied and ./npm-shrinkwrap.json is present,
```

Comment:
This section feels out of place in the `SYNOPSIS`. It's OK to draw people's attention to it with a callout, but it should be in the body of the `DESCRIPTION` with the rest of the docs.